Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM docker.io/debian@sha256:741bae561f5c2261f4cdd535e4fd4c248dec0aafc1b9a1410b3d67ad24571340
```

Comment:
I'd use https://hub.docker.com/r/nginxinc/nginx-unprivileged instead.

Reply:
If you do go down that road, you should be able to drop the nginx.conf completely

Reply:
I was trying to go down that road but got stuck on a dumb problem: which architecture should I pick? I tried to determine which architecture we have picked for debian (`741bae561f5c2261f4cdd535e4fd4c248dec0aafc1b9a1410b3d67ad24571340`) and got stuck there for way too long 😮‍💨

Reply:
I don't think I'll be able to drop the nginx.conf completely though, because I need two ports targeting two different directories (default is 1 folder served on a single port)

Reply:
I call this script `docker-digest`, it's what I use to get the correct checksum for images:
```
#!/bin/sh

set -euf

image="${1:?Missing image name}"

for image in "$@"; do
	echo >&2 "Fetching checksum for '$image'..."

	MANIFEST="$(skopeo inspect docker://"$image" --raw)"
	echo -n "$MANIFEST" | skopeo manifest-digest /dev/stdin
done
```